tests/saliency_metrics/metrics/insertion_deletion/test_insertion_deletion_metric.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+
+from saliency_metrics.metrics.insertion_deletion.insertion_deletion_metric import InsertionDeletion
+from saliency_metrics.models import CUSTOM_CLASSIFIERS
+
+
+@CUSTOM_CLASSIFIERS.register_module()
+class TestNet(nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.features = nn.Sequential(
+            nn.Conv2d(in_channels=3, out_channels=5, kernel_size=3, padding=1),
+            nn.MaxPool2d(2, 2),
+            nn.ReLU(inplace=True),
+        )
+        self.classifier = nn.Sequential(nn.Linear(16 * 16 * 5, 10))
```

Comment:
No need to use such a big head in the test cases. If you use `AdaptiveAvgPool2d(1, 1)`, then `in_features` should be `out_channels` of conv layer. The `out_features` can also be reduced.